test/unit/mock/backend.js

```diff
@@ -73,23 +59,21 @@ responses['client/mget/fail/1'] = responses['client/search/fail/1'];
 function setup( key, cmdCb ){
   function backend( a, b ){
     return {
-      client: {
-        mget: function( cmd, cb ){
-          if( 'function' === typeof cmdCb ){ cmdCb( cmd ); }
-          return responses[key.indexOf('mget') === -1 ? 'client/mget/ok/1' : key].apply( this, arguments );
-        },
-        suggest: function( cmd, cb ){
-          if( 'function' === typeof cmdCb ){ cmdCb( cmd ); }
-          return responses[key].apply( this, arguments );
-        },
-        search: function( cmd, cb ){
-          if( 'function' === typeof cmdCb ){ cmdCb( cmd ); }
-          return responses[key].apply( this, arguments );
-        }
+      mget: function( cmd, cb ){
+        if( 'function' === typeof cmdCb ){ cmdCb( cmd ); }
+        return responses[key.indexOf('mget') === -1 ? 'client/mget/ok/1' : key].apply( this, arguments );
+      },
+      suggest: function( cmd, cb ){
+        if( 'function' === typeof cmdCb ){ cmdCb( cmd ); }
+        return responses[key].apply( this, arguments );
+      },
+      search: function( cmd, cb ){
+        if( 'function' === typeof cmdCb ){ cmdCb( cmd ); }
+        return responses[key].apply( this, arguments );
       }
     };
   }
-  return backend;
+  return backend();
```

Comment:
Can this `backend` function be removed if setup returns the object with responses directly?